certified-connectors/Zoho Mail/readme.md

```diff
@@ -0,0 +1,55 @@
+# Zoho Mail
+
+Zoho Mail connector gives you the power to manage your emails and perform actions such as saving them as drafts, receiving them, and triggering flows when new emails arrive.
+
+## Publisher: Zoho Corporation
+
+## Prerequisites
+
+A verified Zoho Mail account. Click [here](https://zoho.com/zohomail) to create your first account.
+
+## How to get credentials
+
+Zoho Mail API uses OAuth2.0 to authenticate and hence no secondary authentication is required once the account has been set up.
+
+## Supported Operations
+
+The connector supports the following operations:
+
+### Send an email
+
+Send a new email from a specified email address
+
+### Save draft or template 
+
+Save the email content either as a draft or as a template
+
+### Get email content
+
+Get the email content based on message ID
+
+### Search Mail
+
+Search for emails based on search parameters.
+
+## Supported Triggers
+
+The connector supports the following operations:
+
+### New Mail Notification
+
+Receive a new email notification from sender.
+
+### Known Issues and Limitations
+
+* The total email size, including the email headers, body content, inline images, and attachments should not exceed the plan limit.
+* Zoho Mail cannot be used for sending out bulk emails and other emails that violates Zoho Mail usage policy. Click [here](https://www.zoho.com/mail/help/usage-policy.html) for more details.
+
+## FAQ
+
+https://help.zoho.com/portal/en/kb/mail
+
```

Comment:
Hi @zohomail 
Please update readme.md file with all the required section, please refer the template https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md